de/firefox_cloud_services_tos.md

```diff
@@ -1,42 +1,42 @@
-﻿# Firefox Cloud Services: Servicebedingungen
+﻿# Mozilla VPN Servicebedingungen
```

Comment:
This is not VPN! `Mozilla Accounts Terms of Service` is the title in the original doc.

I'm using `Servicebedingungen für Mozilla-Konten` as a stop gap, but please check back with the translator (cc @peiying2).